src/dss/main.py

```diff
@@ -116,5 +118,25 @@ def logs_command(kubeconfig: str, notebook_name: str, print_all: bool, mlflow: b
         get_logs("notebooks", notebook_name, lightkube_client)
 
 
+@main.command(name="purge")
+# FIXME: Remove the kubeconfig param from the create command (and any tests) after
+#  https://github.com/canonical/data-science-stack/issues/37
+@click.option(
+    "--kubeconfig",
+    help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
+)
+def purge_command(kubeconfig: str) -> None:
+    """
+    Removes all notebooks and DSS components.
+    """
+    logger.info("Executing purge command")
+
+    kubeconfig = get_default_kubeconfig(kubeconfig)
+    lightkube_client = get_lightkube_client(kubeconfig)
+    try:
+        purge(lightkube_client=lightkube_client)
+    except Exception:
```

Comment:
Maybe we should also log the exception ... example [here](https://github.com/canonical/data-science-stack/pull/73/files#diff-2f57dc654b74f69ba83e508cc30ef64d0c75f5d95664138c59e693663019d467R160)

Reply:
Pushed this as well